tests/target/type-alias-where-clauses.rs

```diff
@@ -0,0 +1,20 @@
+type Foo
+where
+    A: B,
+    C: D,
+= E;
+
+type Foo
+where
+    A: B,
+    C: D,
+= E
+where
+    F: G,
+    H: I;
+
+type Foo
+    = E
+where
+    F: G,
+    H: I;
```

Comment:
@compiler-errors is there documentation in the style guide that you can link to for this?

I'm wondering why this wouldn't be formatted like:

```rust
type Foo = E
where
    F: G,
    H: I;
```

Reply:
https://github.com/rust-lang/rust/pull/114901

Reply:
Thank you. That's exactly what I was looking for